workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -197,4 +197,19 @@
   end
 
   #Add new rspec test below
+
+  describe '#build_branch' do
+    build_id = 'CozmoOne'
+    branch = 'branch'
+
+     it 'result contains the correct url if triggering a build successfully' do
```

Comment:
Please align "it" same column with variable. (We also update alignment all line in case)